container/kvm/run.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// +build linux
+// +build amd64 arm64 ppc64el
+
+package kvm
+
+import "github.com/juju/utils"
+
+const libvirtUser = "libvirt-qemu"
```

Comment:
I believe this is only the case for ubuntu? if so I would add a comment stating that.